pom.xml

```diff
@@ -691,6 +691,33 @@
                     <argLine>-verbose:gc -Xloggc:"${project.build.directory}/gc.log"</argLine>
                     <argLine>-Duser.timezone=UTC</argLine>
                 </configuration>
+                <executions>
+                    <execution>
+                        <id>default-integration-test</id>
+                        <configuration>
+                            <includes>
+                                <include>**/RestQueryTranslationTest*.java</include>
+                            </includes>
+                            <skip>true</skip>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>multi-thread-test</id>
+                        <phase>integration-test</phase>
+                        <goals>
+                            <goal>test</goal>
+                        </goals>
+                        <configuration>
+                            <includes>
+                                <include>**/RestQueryTranslationTest*.java</include>
+                            </includes>
+                            <forkCount>16</forkCount>
```

Comment:
(Perhaps an obvious statement) As a consideration... if we wanted to have control over the parallelism for RQTTs (or QTTs), we could factor this out a property.  

That let us do something different in Jenkins from local builds.  

Not sure if that's a good idea or overkill.

Reply:
Ultimately, this is the max number of forks, but it's still limited by the number of classes I defined statically, unfortunately, so it can't be parameterized in this way just yet.